cmd/discovery/main.go

```diff
@@ -167,6 +167,9 @@ func ParseConfig(path string) (*DiscoveryConfig, error) {
 	if conf.MasterPort == "" {
 		return nil, errors.New("missing config error, MasterPort must be defined")
 	}
+	if conf.PlayerCount == 0 {
+		return nil, errors.New("missing config error, PlayerCount must be defined")
```

Comment:
The error message sounds misleading as `PlayerCount` is defined, but the given config of 0 (maybe even 1) player(s) defined is invalid.